### PR TITLE
Change the download url

### PR DESCRIPTION
**Pull request by zhangjustin**
## Summary of changes

Change the download url.

## How to Test
```
- src: git@github.com:zhangjustin/ansible-role-python27.git
  scm: git
  name: juwai.python27
  version: 884287408a60f355cce3a16e9ddd42c7313c139d
```
I had test in centos-68 and centos-65, all works.

/fyi: @tjoelsson , @mark-juwai : Please review.

### CODE REVIEW

**Comment by zhangjustin** [comment]
@tjoelsson , It works, but need to remove `--version`.

**Comment by tjoelsson** [comment]
If get-pip script installs pip, you can remove the other task for installing pip

File: tasks/centos6.yml [hunk 1]
```diff
@@ -31,18 +31,12 @@
 
 - name: download setuptools
```
Comment (tjoelsson) [inline tasks/centos6.yml]: download pip install script

File: tasks/centos6.yml [hunk 2]
```diff
@@ -31,18 +31,12 @@
...
   get_url:
-    url: https://bootstrap.pypa.io/ez_setup.py
-    dest: /tmp/ez_setup.py
+    url: https://bootstrap.pypa.io/get-pip.py
+    dest: /tmp/get-pip.py
     timeout: 30
   when: easy_install_check|failed
 
 - name: install setuptools
```
Comment (tjoelsson) [inline tasks/centos6.yml]: install pip, setuptools, and wheel

File: tasks/centos6.yml [hunk 3]
```diff
@@ -31,18 +31,12 @@
...
-  command: "/usr/bin/{{ executable_python }} ez_setup.py --version=19.6.2"
+  command: "/usr/bin/{{ executable_python }} get-pip.py"
   args: { chdir: /tmp }
   when: easy_install_check|failed
```
Comment (tjoelsson) [inline tasks/centos6.yml]: or pip_check|failed

**Review by tjoelsson — CHANGES_REQUESTED** [review]

**Comment by zhangjustin** [comment]
Updated, please review.

File: tasks/centos6.yml [hunk 4]
```diff
@@ -29,14 +29,14 @@
...
     dest: /tmp/get-pip.py
     timeout: 30
   when: easy_install_check|failed
 
-- name: install setuptools
+- name: install pip, setuptools, and wheel
   command: "/usr/bin/{{ executable_python }} get-pip.py"
   args: { chdir: /tmp }
-  when: easy_install_check|failed
+  when: pip_check|failed
```
Comment (tjoelsson) [inline tasks/centos6.yml]: easy_install_check|failed or pip_check|failed

**Review by tjoelsson — APPROVED** [review]

**Review by mark-juwai — APPROVED** [review]

**Pull request merged**